src/rn2xx3.cpp

```diff
@@ -195,6 +212,38 @@ bool rn2xx3::initOTAA(String AppEUI, String AppKey)
   return joined;
 }
 
+
+bool rn2xx3::initOTAA(uint8_t * AppEUI, uint8_t * AppKey, uint8_t * DevEUI)
```

Comment:
Is it really necessary to support keys defined as byte arrays? It will anyway be constants defined in the main program. When copying from the TTN console, it is easier to copy the HEX string, than deciding which array format to copy.